application/ui/src/features/inspect/toolbar/sources/edit-source/edit-source.test.tsx

```diff
@@ -0,0 +1,95 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { useConnectSourceToPipeline } from 'src/hooks/use-pipeline.hook';
+import { TestProviders } from 'src/providers';
+
+import { useSourceMutation } from '../hooks/use-source-mutation.hook';
+import { IpCameraFields } from '../ip-camera/ip-camera-fields.component';
+import { ipCameraBodyFormatter, ipCameraInitialConfig } from '../ip-camera/utils';
+import { IPCameraSourceConfig } from '../util';
+import { EditSource } from './edit-source.component';
+
+vi.mock('../hooks/use-source-mutation.hook');
+vi.mock('src/hooks/use-pipeline.hook');
```

Comment:
I'd prefer to use relative paths here, or at least be consistent among both imports